input-output-hk/mantis
# [ETCM-107] Backport logging configuration

# Description

Backport Logging configuration to allow:
  - logs in JSON (currently disabled, cc @mirkoAlic)
  - setting logs configuration within application.conf
  - automatic adjustment of logs placement depending on datadir setting (important for Luna)

# Important changes introduced

This PR changes logging configuration in a way that may affect deployments. How can I check whether that's true?

# Testing

1. Check whether logs are in same place as previously
2. Check whether logs follow settings under `mantis.logging`
